test/lib/transform.test.js

```diff
@@ -0,0 +1,67 @@
+import * as transform from '../../lib/transform'
+
+var data = [
+ {
+   "a": 17.76,
+   "b": 20.14,
+   "c": 17.05,
+   "d": 17.79
+ },
+ {
+   "a": 19.19,
+   "b": 21.29,
+   "c": 19.19,
+   "d": 19.92
+ },
+ {
+   "a": 20.33,
+   "b": 22.9,
+   "c": 19.52,
+   "d": 21.12
+ },
+ {
+   "a": 20.15,
+   "b": 20.72,
+   "c": 19.04,
+   "d": 19.31
+ },
+ {
+   "a": 17.93,
+   "b": 18.09,
+   "c": 16.99,
+   "d": 17.01
```

Comment:
Why not to use simple numbers like 10, 20, 50 ....? would be easier to calculate the expected output